docs/content/about-iota/iota-architecture/iota-security.mdx

```diff
@@ -22,54 +24,54 @@ The set of validators changes periodically and is [determined by the amount of l
 
 ### Choosing Validators
```

Comment:
Move this subsection up. Rename it to reflect delegated PoS consensus mechanism.